### ISSUE TITLE
Switch over MUI components to new Styled format so we can upgrade

### ISSUE THREAD

**Issue by nabramow** [Couchers-org/couchers#5299]
**NOTE:** Best to do a few components at a time, then open a PR. Then we can more easily test those few changes and make sure nothing broke.

------

Before we can upgrade MUI to v6 and above, we need to finish changed over our style from `makeStyles` to the new `Styled`.

If you search `Styled` you should find many examples already done.

This ticket should be relatively straight-forward though it will be a lot of changes. Feel free to do PRs in sections, as it makes it easier to test, i.e. do a group of 10 components in 1 PR, open it. Do another PR with another 10.

** Make sure to import `styled` from `@mui/material` (it comes up from other packages in the suggestions).**

Please name the new styled components starting with `Styled`, i.e. `StyledAlert`, `StyledCollapse`, etc..

Here is the migration from JSS guide from MUI if it helps:
https://mui.com/material-ui/migration/migrating-from-jss/

Old way:

```javascript
const useStyles = makeStyles((theme) => ({
  root: {
    marginBottom: theme.spacing(2),
  },
}));
```

New way:

```javascript
import { theme } from "theme";

const StyledAlert = styled("div", {
    marginBottom: theme.spacing(2),
});
```
